pkg/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -118,31 +118,11 @@ func (h *Helper) CheckAndPull(image string, out io.Writer) error {
 	}
 	glog.V(5).Infof("Image %q not found. Pulling", image)
 	fmt.Fprintf(out, "Pulling image %s\n", image)
```

Comment:
when you go to resolve the logging issue in oc cluster up, you should change this to a glog call also.